lib/everypolitician.rb

```diff
@@ -145,12 +145,16 @@ def csv_url
     end
 
     def csv
-      CSV.parse(open(csv_url).read, headers: true, header_converters: :symbol, converters: nil)
+      @csv ||= CSV.parse(open(csv_url).read, headers: true, header_converters: :symbol, converters: nil)
     end
 
     def [](key)
       raw_data[key]
     end
+
+    def unique_people
+      csv.map(&:to_hash).uniq { |person| person[:id] }
```

Comment:
I'm a little worried that there's still a mismatch between what's being asked for here, and what's being returned, that could be confusing in some cases, and possibly bug-inducing in others. 

e.g. if someone starts asking any of these 'People' for things like a `start_date` or `end_date` odd things will happen.